Makefile.in

```diff
@@ -52,7 +52,14 @@ endif
 export PATH AWK SED
 
 MULTILIB_FLAGS := @multilib_flags@
+MULTILIB_GEN := @multilib_gen@
+ifeq ($(MULTILIB_GEN),)
 NEWLIB_MULTILIB_NAMES := @newlib_multilib_names@
+GCC_MULTILIB_FLAGS := $(MULTILIB_FLAGS)
+else
+NEWLIB_MULTILIB_NAMES := $(shell echo $(MULTILIB_GEN) | $(SED) 's/;/\n/'| $(AWK) '{split($0,a,"-"); printf "%s-%s ", a[1],a[2]}')
```

Comment:
I'm seeing errors

gawk: cmd. line:1: {split(,a,"-"); printf "%s-%s ", a[1],a[2]}
gawk: cmd. line:1:        ^ syntax error

Looks like a quoting problem.  I can fix it by changing $0 to $$0 but I'm not sure if that is the best solution.

Reply:
> Looks like a quoting problem. I can fix it by changing $0 to $$0 but I'm not sure if that is the best solution.

Seems like $$0 is right, but so weird, it work on my ubunut 18.04 server...